cli/command/formatter/formatter.go

```diff
@@ -50,10 +57,12 @@ type Context struct {
 
 func (c *Context) preFormat() {
 	c.finalFormat = string(c.Format)
-
 	// TODO: handle this in the Format type
-	if c.Format.IsTable() {
+	switch {
+	case c.Format.IsTable():
 		c.finalFormat = c.finalFormat[len(TableFormatKey):]
+	case c.Format.IsJSON():
+		c.finalFormat = jsonFormat
 	}
 
 	c.finalFormat = strings.Trim(c.finalFormat, " ")
```

Comment:
I'll have a look at slightly cleaning this up in a follow-up; looks like there's some steps here that may not be needed in all cases